nix/packages.nix

```diff
@@ -249,6 +249,7 @@ in rec {
     gcloud = pkgs.google-cloud-sdk;
     bq     = gcloud;
     gsutil = gcloud;
+    docker-credential-gcr = pkgs.docker-credential-gcr;
```

Comment:
It would be nice to add comment documenting what this is used for. Otherwise, someone might end up removing it again, e.g., on a nixpkgs upgrade because they think if our CI builds don’t use it, it must be unnecessary.

Reply:
Ah, but the point is it would be used by CI; that's the whole reason for putting it there. Though I take the feedback that it's not used by the regular, on-PR build so probably worth pointing that out.